specification/network/resource-manager/Microsoft.Network/stable/2020-03-01/expressRouteProviderPort.json

```diff
@@ -0,0 +1,206 @@
+{
+	"swagger": "2.0",
+	"info": {
+		"title": "NetworkManagementClient",
+		"description": "The Microsoft Azure Network management API provides a RESTful set of web services that interact with Microsoft Azure Networks service to manage your network resources. The API has entities that capture the relationship between an end user and the Microsoft Azure Networks service.",
+		"version": "2020-03-01"
+	},
+	"host": "management.azure.com",
+	"schemes": [
+		"https"
+	],
+	"consumes": [
+		"application/json"
+	],
+	"produces": [
+		"application/json"
+	],
+	"security": [
+		{
+			"azure_auth": [
+				"user_impersonation"
+			]
+		}
+	],
+	"securityDefinitions": {
+    "azure_auth": {
+			"type": "oauth2",
+			"authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+			"flow": "implicit",
+			"description": "Azure Active Directory OAuth2 Flow.",
+			"scopes": {
+				"user_impersonation": "impersonate your user account"
+			}
+		}
+	},
+  "paths": {
+    "/subscriptions/{subscriptionId}/providers/Microsoft.Network/expressRouteProviderPorts": {
+      "get": {
+        "tags": [
+          "ExpressRouteProviderPorts"
+        ],
+        "operationId": "ExpressRouteProviderPorts_List",
+        "description": "Retrieves all the ExpressRouteProviderPorts in a subscription.",
+        "parameters": [
+          {
+            "$ref": "./network.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "./network.json#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Request successful. The operation returns a list of ExpressRouteProviderPort resources. If there are no ports for the provider, an empty list is returned.",
+            "schema": {
+              "$ref": "#/definitions/ExpressRouteProviderPortListResult"
+            }
+          },
+          "default": { 
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "./network.json#/definitions/CloudError"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "ExpressRouteProviderPortList": {
+            "$ref": "./examples/ExpressRouteProviderPortList.json"
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.Network/expressRouteProviderPorts?location={locationName}": {
+      "get": {
+        "tags": [
+          "ExpressRouteProviderPorts"
```

Comment:
Looks like you only have defined the list at subscription scope. Is this resource defined elsewhere at the resource group or subscription scope ? 


You need to have the resources defined with atleast a Get at the individual resource scope.

Reply:
done